src/features/script-load.js

```diff
@@ -2,8 +2,8 @@
  * Script instantiation loading
  */
 import { hasDocument } from '../common.js';
-import { systemJSPrototype } from '../system-core.js';
 import { errMsg } from '../err-msg.js';
+import { systemJSPrototype } from '../system-core.js';
```

Comment:
Please move this import statement back to where it was, so that it doesn't show a change.

Reply:
Done, sorry about that it was my vscode plugin to arrange imports